options/locale/locale_en-US.ini

```diff
@@ -3264,6 +3264,14 @@ container.layers = Image Layers
 container.labels = Labels
 container.labels.key = Key
 container.labels.value = Value
+debian.registry = Setup this registry from the command line:
+debian.registry.info = Choose $distribution and $component from the list below.
+debian.install = To install the package, run the following command:
+debian.documentation = For more information on the Debian registry, see <a target="_blank" rel="noopener noreferrer" href="https://docs.gitea.io/en-us/packages/debian/">the documentation</a>.
```

Comment:
https://github.com/go-gitea/gitea/pull/22854#discussion_r1180825472

Was a decision reached? I also notice this URL has a trailing slash while the one below hasn't, is it intentional?

Still, I think we should remove these links from translation. We certainly don't want to force re-translation when the URL changes. Many translations take months to complete after all, some never complete.

Reply:
So, I wonder how we should translate this without HTML, maybe something like this and change the opening tag after translation via string replacement?

```
For more information on the Debian registry, see <a>the documentation</a>
```

Reply:
Brace syntax may depend on https://github.com/go-gitea/gitea/issues/23863. Could also do something like this and fill the attributes during rendering via string replacement of `<a>`:

```
For more information on the Debian registry, see  <a>documentation</a>
```

Reply:
Or maybe even circumvent the whole issue:

```
For more information on the Debian registry, see the documentation:
```

Then, append `<a>Link</a>` after it.

Reply:
`For more information on the Debian registry, see the documentation:`

It seems more flexible.

Reply:
But that's not how links should work?

> For more information on the Debian registry, see the documentation: [Documentation](http://gitea.io)

vs

> For more information on the Debian registry, see [the documentation](http://gitea.io).

Reply:
Hmm I guess I can accept if we just interpolate the URL until we have a better system to support this. This is in use already at `install.docker_helper`:

```
For more information on the Debian registry, see <a target="_blank" rel="noopener noreferrer" href="%s">the documentation</a>`